docs/Style-Guide.md

```diff
@@ -0,0 +1,180 @@
+
+### Get straight to the point
+
+Give your readers a clear and precise action to take. Start with what is most important. This way, you can help them find what they need faster. Mostly, readers tend to read the first content on a page, and many will not scroll further.
+
+**Example**
+
+Less like this: Colons are very important to register a parametric path. It lets the framework know a there is a new parameter created. You can place the colon before the parameter name for the parametric path can be created.
+
+More Like this: To register a parametric path, put a colon before the parameter name. Using colon lets the framework know it is a parametric path and not a wild card.
+
+### Avoid adding video or image content 
+
+
+Do not add videos or screenshots in the documentation. It is easier to keep under version control. Videos and images will eventually end up becoming outdated as new updates keep developing. Instead, make a referral link or a youtube video. You can add links by using `[Title](www.websitename.com)` in the markdown.
+
+**Example**
+
+```
+To learn more about hooks, See [Fastify hooks](https://www.fastify.io/docs/latest/Hooks).
+```
+
+Result:
+>To learn more about hooks, See [Fastify hooks](https://www.fastify.io/docs/latest/Hooks/).
+
+
+
+### Avoid plagiarism
+
+Make sure you evade copying other people's work, keep it as original as possible. You can learn from what they've done and also reference where it's from if you used a particular quote from their work.
+
+
+## Word Choice
+
+
+There are a few things you need to use and avoid when writing your documentation which will improve readability for readers and also make documentations neat, direct and clean all through your writing process.
+
+
+### When to use the second person "you" as the pronoun
+
+When writing articles or step-by-step guides, your content should communicate directly to readers who is the second person ("you") addressed. It is easier to give them direct instruction on what to do on a particular topic. To see an example, visit the [Plugins-guide.md](https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Plugins-Guide.md) page on Github. 
+
+**Example**
+
+Less Like this: we can use the following plugins.
+
+More like this: You can use the following plugins.
+
+> According to [Wikipedia](#), ***You*** is usually a second person pronoun. Also, used to refer to an indeterminate person, as a more common alternative to a very formal indefinite pronoun.
+
+## When to avoid second person "you" as the pronoun
+
+One of the main rules of formal writing such as references, synthesis and also in API documentation is to avoid second person ("you"). It should not directly address the user even when the word "you" is almost unavoidable when writing, but there are ways to avoid second person. Below are some alternative examples you can use to evade them. 
```

Comment:
```suggestion
One of the main rules of formal writing such as reference documentation, or API documentation, is to avoid second person ("you"), avoiding directly addressing the reader.
```

There is only one example, so saying that "below are some examples" is not helpful.

Frankly, this section could be a document unto itself, e.g. https://docs.microsoft.com/en-us/style-guide/developer-content/reference-documentation